### PR TITLE
feat: Support Prometheus 3.0

### PR DESCRIPTION
**Pull request by SuperQ**
Update prometheus role for 3.0.0.
* Update argument_specs.yml to reflect support of 3.x.
* Skip console templates on 3.x.

Fixes: https://github.com/prometheus-community/ansible/issues/460

### CODE REVIEW

**Comment by SuperQ** [comment]
I went over https://prometheus.io/docs/prometheus/3.0/migration/ and didn't find anything incompatible in our role.

File: roles/prometheus/tasks/configure.yml [hunk 1]
```diff
@@ -58,6 +60,7 @@
   notify:
     - restart prometheus
   become: true
+  when: prometheus_version is version_compare('3.0.0', '<')
```
Comment (gardar) [inline roles/prometheus/tasks/configure.yml:63]: Should probably be just version and not version_compare, afaik the version_compare name has been deprecated

Comment (SuperQ) [reply roles/prometheus/tasks/configure.yml:63]: Ahh, that was copy-pasta from other roles. Will update it.

**Comment by SuperQ** [comment]
Found and fixed a couple of minor issues.
* It doesn't come with console templates anymore, but still supports them.
* It renamed the agent feature flag to just a flag.

**Review by gardar — APPROVED** [review]
LGTM!

**Pull request merged**